sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_events_v1/query.sql

```diff
@@ -0,0 +1,1471 @@
+-- Query for fenix_derived.feature_usage_events_v1
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+DECLARE start_date DATE DEFAULT "2021-01-01";
+
+DECLARE end_date DATE DEFAULT current_date;
+
+WITH dau_segments AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    COUNT(DISTINCT client_info.client_id) AS dau
+  FROM
+    `mozdata.fenix.events_unnested`
+      --AND channel = 'release'
+  WHERE
+    DATE(submission_timestamp) >= '2021-01-01'
+  GROUP BY
+    1
+),
+product_features AS (
+  SELECT
+    client_info.client_id,
+    DATE(submission_timestamp) AS submission_date,
+    /*Logins*/
+    --autofill
+    CASE
```

Comment:
Would this be a bit more readable if we used an `if` instead of a `CASE`? 

For example:
`IF(event_category = 'logins' AND event_name = 'password_detected', 1, 0) AS autofill_password_detected_logins,`?